src/apps/explorer/components/OrderWidget/OrderDetails.tsx

```diff
@@ -46,10 +35,8 @@ const Table = styled(SimpleTable)`
         }
       }
 
-      height: 5rem;
-
-      :not(:first-of-type) {
-        padding-left: 2.5rem;
+      &:last-of-type {
+        color: ${FONT_COLOR_DESCRIPTION};
```

Comment:
same here, theme

Reply:
Fixed.

Reply:
@biocom I marked some as fixed. We usually mark as resolved ourselves if we clearly address the issue